### DISCUSSION TITLE
[BUG] Error 1. Retrying in five minutes.

### DISCUSSION THREAD

**Discussion by Edison5151** [Chia-Network/chia-blockchain#3178] [General]
my plots stuck at 32% and this error message appears:

Only wrote 0 of 1048570 bytes at offset 324008130 to "E: \\ temp \\ plot-k32-2021-04-28-16-26-cb227fce81bae36f26bb48225c0d4b0fa5448cf99f99cd6e0c83ee28789.dp .t2.sort_bucket_081.tmp "with length 324009984. Error 1. Retrying in five minutes.

Can someone help me?

**Comment by maomingyang1314** [comment]
Check the disk capacity, whether there is a problem

**Comment by Edison5151** [comment]
> Verifique a capacidade do disco, se há um problema

I have a 256gb ssd, I used two directory for temporary files to any problem?

**Comment by maomingyang1314** [comment]
In other words, you created two directories under the 256G ssd?

**Comment by maomingyang1314** [comment]
I use a 1TB SSD for drawing.

**Comment by Edison5151** [comment]
no, i used a 256gb ssd and another 500gb hd to write

**Comment by maomingyang1314** [comment]
You should use a 500G disk as the zero-time file directory, and use a 256G SSD as the target address

**Comment by maomingyang1314** [comment]
256GB disk may not be enough

**Comment by Edison5151** [comment]
thanks, i will buy a 500gb ssd

**Comment by maomingyang1314** [comment]
It is recommended to buy a 1TB version, which can support drawing three pictures at the same time

**Comment by maomingyang1314** [comment]
SN750 1TB version or PM981a 1TB version

**Comment by hieuhdt** [comment]
@maomingyang1314 I use 2TB SSD and 6TB HDD to plot but I still meet the same issue. Sometimes it stopped 1%, 11%, 32%,...
Can you help me?

P/s:
OS: Ubuntu 20
CPU: Intel core i5 5200U
Ram: 16GB

**Reply by PsProg** [reply]
@thienhieu123  did you solve the problem?
